naplib/io/fileio.py

```diff
@@ -216,6 +216,9 @@ def load(filename):
     with open(filename, 'rb') as inp:
         output = pickle.load(inp)
 
+    if 'labels_data' in output:
+        output['labels_data'] = np.array(output['labels_data'])
+
```

Comment:
I don't think this should be added here. This is just a generic load function which can load from anything you pickled. But this assumes the thing you loaded is a dict

Reply:
Good point. I moved it to `process_ieeg`.